zebra-chain/src/transaction/sighash.rs

```diff
@@ -48,6 +52,23 @@ impl HashType {
     }
 }
 
+/// A Signature Hash (or SIGHASH) as specified in
+/// https://zips.z.cash/protocol/protocol.pdf#sighash
+#[derive(Copy, Clone, Eq, PartialEq, Hash, Serialize, Deserialize, Debug)]
```

Comment:
FYI the `std::hash::Hash` trait is '...able to be hashed with an instance of Hasher', which is true here but just making sure that's clear